ifaddr/_shared.py

```diff
@@ -143,25 +155,25 @@ class sockaddr_in6(ctypes.Structure):
 
 else:
 
-    class sockaddr(ctypes.Structure):
+    class sockaddr(ctypes.Structure):  # type: ignore
         _fields_= [('sa_familiy', ctypes.c_uint16),
                    ('sa_data', ctypes.c_uint8 * 14)]
 
-    class sockaddr_in(ctypes.Structure):
+    class sockaddr_in(ctypes.Structure):  # type: ignore
         _fields_= [('sin_familiy', ctypes.c_uint16),
                    ('sin_port', ctypes.c_uint16),
                    ('sin_addr', ctypes.c_uint8 * 4),
                    ('sin_zero', ctypes.c_uint8 * 8)]
 
-    class sockaddr_in6(ctypes.Structure):
+    class sockaddr_in6(ctypes.Structure):  # type: ignore
         _fields_= [('sin6_familiy', ctypes.c_uint16),
                    ('sin6_port', ctypes.c_uint16),
                    ('sin6_flowinfo', ctypes.c_uint32),
                    ('sin6_addr', ctypes.c_uint8 * 16),
                    ('sin6_scope_id', ctypes.c_uint32)]
 
```

Comment:
Why re those '#type: ignore' annotations above needed? (curious)

Reply:
`Mypy` expects types to have a single definition (and ignores platform-dependent `if` statements).